FAQ.md

```diff
@@ -85,11 +85,29 @@ Here are some suggestions for when you could use `dep` or `go get`:
 -[@sdboyer in #376](https://github.com/golang/dep/issues/376#issuecomment-294045873)
 
 ## Why is `dep` ignoring a version constraint in the manifest?
-Only your project's directly imported dependencies are affected by a `dependencies` entry
+Only your project's directly imported dependencies are affected by a `constraint` entry
 in the manifest. Transitive dependencies are unaffected.
 
 Use an `overrides` entry for transitive dependencies.
 
+By default, when you specify a version without an operator, such as `~` or `=`,
+`dep` automatically adds a caret operator, `^`. The caret operator pins the
+left-most non-zero digit in the version. For example:
+```
+^1.2.3 means 1.2.3 <= X < 2.0.0
+^0.2.3 means 0.2.3 <= X < 0.3.0
+^0.0.3 means 0.0.3 <= X < 0.0.4
```

Comment:
i know this is what rust does, but i don't think this is what we actually do. I believe that `^0.0.3` is equivalent to `0.0.3 <= X < 0.1.0`.